tensor_sum.ipynb

```diff
@@ -0,0 +1,180 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = keras.Sequential([\n",
+    "    keras.layers.Flatten(input_shape=(2,)),\n",
+    "\tkeras.layers.Dense(20, activation=tf.nn.relu),\n",
+    "    keras.layers.Dense(20, activation=tf.nn.relu),\n",
+    "    keras.layers.Dense(1)\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.compile(optimizer='adam', \n",
+    "              loss='mse',\n",
+    "              metrics=['accuracy'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Train on 9 samples\n",
+      "Epoch 1/10\n",
+      "9/9 [==============================] - 0s 28ms/sample - loss: 60.8714 - accuracy: 0.0000e+00\n",
+      "Epoch 2/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 56.1792 - accuracy: 0.0000e+00\n",
+      "Epoch 3/10\n",
+      "9/9 [==============================] - 0s 914us/sample - loss: 51.4382 - accuracy: 0.0000e+00\n",
+      "Epoch 4/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 47.9816 - accuracy: 0.0000e+00\n",
+      "Epoch 5/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 44.2236 - accuracy: 0.0000e+00\n",
+      "Epoch 6/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 40.9023 - accuracy: 0.0000e+00\n",
+      "Epoch 7/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 38.3334 - accuracy: 0.0000e+00\n",
+      "Epoch 8/10\n",
+      "9/9 [==============================] - 0s 932us/sample - loss: 35.1447 - accuracy: 0.0000e+00\n",
+      "Epoch 9/10\n",
+      "9/9 [==============================] - 0s 1ms/sample - loss: 32.3164 - accuracy: 0.0000e+00\n",
+      "Epoch 10/10\n",
+      "9/9 [==============================] - 0s 892us/sample - loss: 29.6585 - accuracy: 0.0000e+00\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<tensorflow.python.keras.callbacks.History at 0x7f81c47a4668>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.fit(X_train, y_train, batch_size=1, epochs=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2/2 - 0s - loss: 29.9146 - accuracy: 0.0000e+00\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[29.914562225341797, 0.0]"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.evaluate(X_test, y_test, verbose=2)"
```

Comment:
it would be great to plot the loss as well to see if it converges or there is more to improve